librarian_packages/hera_librarian/tests/test_base_store.py

```diff
@@ -0,0 +1,183 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+"""Test code in hera_librarian/store.py
+
+"""
+
+from __future__ import print_function, division, absolute_import
+import pytest
+import os
+import json
+from hera_librarian import base_store, RPCError
+
+from . import ALL_FILES, filetypes, obsids, md5sums, pathsizes
+
+
+@pytest.fixture
+def local_store():
+    return base_store.BaseStore("local_store", "/tmp", "localhost")
+
+
+def test_path(local_store):
+    # test that store path is prepended
+    assert local_store._path("my_dir") == "/tmp/my_dir"
+
+    # test passing in an absolute path
+    with pytest.raises(ValueError):
+        local_store._path("/tmp/my_dir")
+
+    return
+
+
+def test_ssh_slurp(local_store):
+    # test simple command
+    assert local_store._ssh_slurp("echo hello world") == "hello world\n"
+
+    # try a bogus command
+    with pytest.raises(RPCError):
+        local_store._ssh_slurp("foo")
+
+    return
+
+
+def test_copy_to_store(tmpdir, local_store):
+    # make a fake file in our tmpdir
+    tmpfile = os.path.join(str(tmpdir), "my_file.txt")
```

Comment:
Super nitpicky, especially for a test, but I think it's good in variables to reserve "file" for open file handles, and to make a point of referring to the textual names of where files live as *paths*.

Reply:
Good point, changed it.